freeipa/src/main/java/com/sequenceiq/freeipa/converter/image/ImageConverter.java

```diff
@@ -16,8 +16,7 @@
     public com.sequenceiq.cloudbreak.cloud.model.Image convert(Image source) {
         com.sequenceiq.cloudbreak.cloud.model.Image image =
                 new com.sequenceiq.cloudbreak.cloud.model.Image(source.getImageName(),
-                        Map.of(InstanceGroupType.GATEWAY, Optional.ofNullable(source.getUserdata()).orElse(""),
-                                InstanceGroupType.CORE, Optional.ofNullable(source.getUserdata()).orElse("")),
+                        Map.of(InstanceGroupType.GATEWAY, Optional.ofNullable(source.getUserdata()).orElse("")),
```

Comment:
do we really want to remove the CORE part here ?

Reply:
This was done to simplify the FreeIPA templates. I discussed this with @lacikaaa and @keyki

Reply:
👍